### PR TITLE
docker-machine-driver-xhyve: fix #6678 for installs outside of default location

### LINKED ISSUE #6678: docker-machine-driver-xhyve failed to build on 10.10.5
### Bug reports:

New version of `docker-machine-driver-xhyve`, `0.3.0`, fails to build with the following message:

```
==> make build
Last 15 lines from /Users/rag/Library/Logs/Homebrew/docker-machine-driver-xhyve/04.make:
  "_ev_loop_destroy", referenced from:
      _lwt_libev_stop in liblwt-unix_stubs.a(lwt_libev_stubs.o)
  "_ev_loop_new", referenced from:
      _lwt_libev_init in liblwt-unix_stubs.a(lwt_libev_stubs.o)
  "_ev_run", referenced from:
      _lwt_libev_loop in liblwt-unix_stubs.a(lwt_libev_stubs.o)
  "_ev_set_invoke_pending_cb", referenced from:
      _lwt_libev_init in liblwt-unix_stubs.a(lwt_libev_stubs.o)
  "_ev_timer_start", referenced from:
      _lwt_libev_timer_init in liblwt-unix_stubs.a(lwt_libev_stubs.o)
  "_ev_timer_stop", referenced from:
      _lwt_libev_timer_stop in liblwt-unix_stubs.a(lwt_libev_stubs.o)
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make: *** [bin/docker-machine-driver-xhyve] Error 2
```

https://gist.github.com/mtpereira/488d34625ae4639812d0c21aa41bd1cc

### PR DESCRIPTION
**Pull request by cdelacroix**
The project Makefile contains hardcoded
`LIBEV_FILE ?= /usr/local/lib/libev.a`
so it works only with the default install location

The patch sets the LIBEV_FILE value from the libev Formula hash

### CODE REVIEW

**Comment by JCount** [comment]
@BrewTestBot test this please

File: Formula/docker-machine-driver-xhyve.rb [hunk 1]
```diff
@@ -52,6 +52,7 @@ def install
       go_ldflags = "-w -s -X 'github.com/zchee/docker-machine-driver-xhyve/xhyve.GitCommit=Homebrew#{git_hash}'"
       ENV["GO_LDFLAGS"] = go_ldflags
       ENV["GO_BUILD_TAGS"] = build_tags
+      ENV["LIBEV_FILE"] = "#{Formula["libev"].lib}/libev.a"
```
Comment (JCount) [inline Formula/docker-machine-driver-xhyve.rb]: This should use `ENV["LIBEV_FILE"] = "#{Formula["libev"].opt_lib}/libev.a"`

**Comment by cdelacroix** [comment]
@JCount updated, thanks for the review

**Comment by nijikon** [comment]
You are awesome, thanks for your contribution! :tada:

**Pull request closed without merging**